include/z64transition.h

```diff
@@ -15,16 +15,58 @@ typedef struct {
     /* 0x20 */ s32   (*isDone)(void* transition);
 } TransitionInit; // size = 0x24
 
+typedef struct {
+    union {
+        struct {
+    /* 0x00 */ u32 count : 8;
+    /* 0x00 */ uintptr_t addr : 24;
+        } load;
+    /* 0x00 */ u32 loadInfo;
+    };
+    /* 0x04 */ void* vramStart;
+    /* 0x08 */ void* vramEnd;
+    /* 0x0C */ uintptr_t vromStart;
+    /* 0x10 */ uintptr_t vromEnd;
+    /* 0x14 */ TransitionInit* initInfo;
+    /* 0x18 */ size_t size;
+} TransitionOverlay;
+ 
+typedef struct {
+    /* 0x000 */ s16 transitionType;
+    /* 0x002 */ s8 fbdemoType;
+    /* 0x003 */ char unk03[0x5];
+    /* 0x008 */ s32 instanceData;
+    /* 0x00C */ char unk0C[0x224];
+    /*
+    union {
+       TransitionFade fade;
+       TransitionCircle circle;
+       TransitionTriforce triforce;
+       TransitionWipe wipe;
+    } instanceData;
+    */
```

Comment:
What's the point of this commented out union?

Reply:
Could add a TODO to uncomment this when the transitions are done.